css/main.css

```diff
@@ -480,11 +480,134 @@ footer {
 }
 
 @media screen and (min-width: 1024px) {
-  .description-what-is-content {
+  .description-content {
     padding: 0px 250px 90px;
   }
 
-  .description-why-join-content {
+  .features-content {
     padding: 0px 250px 80px 210px;
   }
 }
+
+/***********************/
+/* EMAIL CAPTURE FORM */
+/***********************/
+
+.section-cta {
+  padding: 4.8rem 15.4rem;
+}
+
+.cta {
+  display: grid;
+  border-radius: 9px;
+  align-items: center;
+  justify-content: center;
+
+  box-shadow: 0 1.2rem 2.4rem rgba(0, 0, 0, 0.1);
+}
+
+.cta-text-box {
+  padding: 3.6rem 4.8rem 4.8rem;
+  color: #041484;
+}
+
+.cta .heading-secondary {
+  font-size: 2.4rem;
+  margin-bottom: 1.8rem;
+  color: inherit;
+}
+
+.cta-text {
+  font-size: 1.2rem;
+  line-height: 1.8;
+  margin-bottom: 2.4rem;
+}
+
+.cta-form {
+  display: grid;
+  grid-template-columns: 60% 30%;
+  column-gap: 3.6rem;
+  row-gap: 1.2rem;
+}
+
+.cta-form label {
+  display: block;
+  font-size: 1.2rem;
+  font-weight: 500;
+  margin-bottom: 1.2rem;
+}
+
+.cta-form input {
+  width: 100%;
+  padding: 1.2rem;
+  font-size: 1.2rem;
+  font-family: inherit;
+  border: none;
+  border-radius: 9px;
+
+  color: #777;
+  background-color: #edf2ff;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.1);
+}
+
+.cta-form input:focus {
+  outline: 3px solid #db09795d;
+}
+
+.btn-cta {
+  display: inline-block;
+  font-size: 1.2rem;
+  font-weight: 600;
+  text-transform: uppercase;
+  padding: 1.2rem 2.15rem;
+  border-radius: 9px;
+  border: none;
+  cursor: pointer;
+  font-family: inherit;
+  text-decoration: none;
+  align-items: center;
+  justify-items: center;
+  align-self: end;
```

Comment:
these three lines don't make sense, they can be used for a items which have a display flex, and we don't need to have them.